app/middlewares/fields_validation.js

```diff
@@ -2,12 +2,18 @@ const { validationResult } = require('express-validator');
 
 const { fieldValidationError } = require('../errors');
 
+const buildErrorMessage = errors =>
+  errors
+    .array()
+    .map(error => error.msg)
+    .join('; ');
```

Comment:
Is the `array` function necessary here?

Reply:
Yes, because `errors` is not an array, it is an express-validator object that contains an array with the errors. When calling array(), it returns this array.